packages/perseus/src/widgets/graded-group-set.jsx

```diff
@@ -33,35 +33,41 @@ type IndicatorsProps = {|
     onChangeCurrentGroup: (groupNumber: number) => void,
 |};
 
-// TODO(jeremy): This indicator panel is not accessible for keyboard
-// navigation. Add in keyboard handling and tab indexes to make accessible.
 class Indicators extends React.Component<IndicatorsProps> {
-    render(): React.Node {
-        const items = [];
-        for (let i = 0; i < this.props.numGroups; i++) {
-            items.push(
-                <div
-                    role="button"
-                    aria-label={i18n._("Skip to %(title)s", {
-                        title: this.props.groupTitles[i],
-                    })}
-                    key={i}
-                    className={css(
-                        styles.indicator,
-                        i === this.props.currentGroup &&
-                            styles.selectedIndicator,
-                    )}
-                    onClick={() => this.props.onChangeCurrentGroup(i)}
-                >
-                    {i === this.props.currentGroup && (
-                        <span className={css(a11y.srOnly)}>
-                            {i18n._("Current")}
-                        </span>
-                    )}
-                </div>,
-            );
+    handleKeyDown = (e: SyntheticKeyboardEvent<>, i: number) => {
+        if (e.key === "Enter") {
+            this.props.onChangeCurrentGroup(i);
         }
-        return <div className={css(styles.indicatorContainer)}>{items}</div>;
+    };
+
+    render(): React.Node {
+        return (
+            <ul className={css(styles.indicatorContainer)}>
+                {this.props.groupTitles.map((title, i) => (
+                    <li
+                        role="button"
+                        aria-label={i18n._("Skip to %(title)s", {
+                            title: this.props.groupTitles[i],
```

Comment:
```suggestion
                            title,
```
Since we already have access to title, could this work?

Reply:
Yes! I didn't look too closely at the guts of this component.